versions/3.0.md

```diff
@@ -1,22 +1,22 @@
 # OpenAPI Specification
-## (fka Swagger RESTful API Documentation Specification)
 
-#### Version 2.0
+#### Version 3.0
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://www.ietf.org/rfc/rfc2119.txt).
 
-The Swagger specification is licensed under [The Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0.html).
+The OpenAPI Specification is licensed under [The Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0.html).
 
 ## Introductions
 
-Swagger™  is a project used to describe and document RESTful APIs.
+The OpenAPI Specification is a project used to describe and document RESTful APIs.
 
-The Swagger specification defines a set of files required to describe such an API. These files can then be used by the Swagger-UI project to display the API and Swagger-Codegen to generate clients in various languages. Additional utilities can also take advantage of the resulting files, such as testing tools.
+The OpenAPI Specification defines a set of files required to describe such an API. These files can then be used by the Swagger-UI project to display the API and Swagger-Codegen to generate clients in various languages. Additional utilities can also take advantage of the resulting files, such as testing tools.
```

Comment:
While this is not vendor neutral, there is no existing UI to reference.  Will update it later when tooling proceeds in a neutral fashion.